src/Microsoft.Framework.Localization/ResourceManagerStringLocalizer.cs

```diff
@@ -94,6 +94,28 @@ public IStringLocalizer WithCulture(CultureInfo culture)
         }
 
         /// <summary>
+        /// Returns all strings in the current culture.
+        /// </summary>
+        /// <returns>The strings.</returns>
+        public virtual IEnumerable<LocalizedString> GetAllStrings() => GetAllStrings(CultureInfo.CurrentUICulture);
+
+        /// <summary>
+        /// Returns all strings in the specified culture.
+        /// </summary>
+        /// <param name="culture">The <see cref="CultureInfo"/> to get strings for.</param>
+        /// <returns>The strings.</returns>
+        protected IEnumerable<LocalizedString> GetAllStrings([NotNull] CultureInfo culture)
+        {
+            var resourceNames = GetResourceNamesFromCultureHierarchy(culture);
+
+            foreach (var name in resourceNames)
+            {
+                var value = GetStringSafely(name, culture);
+                yield return new LocalizedString(name, value ?? name, resourceNotFound: value == null);
```

Comment:
According to the function definition https://github.com/aspnet/Localization/blob/dev/src/Microsoft.Framework.Localization.Abstractions/LocalizedString.cs#L30  I don't think that we need the named parameter ```resourceNotFound``` while you didn't change the order of the parameters